R/ChromBackend-functions.R

```diff
@@ -1,43 +1,17 @@
-.valid_chrom_backend_data_storage <- function(x) {
-    if (anyNA(x))
-        return("'NA' values in dataStorage are not allowed.")
-    NULL
-}
-
-#' Helper function that matches `x` against `mz` (using the `closest` function)
-#' and returns the indices of `x` that match any of the values in `mz`. The
-#' function takes care of sorting `x` and `mz` and deals also with missing
-#' values.
-#'
-#' @return `integer` with the indices of values in `x` that are not `NA` and
-#'     are matching any of the values in `mz` given `ppm` and `tolerance`.
-#'
-#' @noRd
-#'
-#' @importFrom MsCoreUtils common
-#'
-#' @author Sebastian Gibb, Johannes Rainer
-.values_match_mz <- function(x, mz, ppm = 20, tolerance = 0) {
```

Comment:
This is used later on for the filter function that you implemented. I kept it on another file locally, but I only wanted things related to what is implemented now to be in this PR. So it will probably come back when I start implementing filter functions :)

Reply:
Thanks for the clarification!